packages/lumx-react/src/components/expansion-panel/ExpansionPanel.tsx

```diff
@@ -38,14 +31,8 @@ interface ExpansionPanelProps extends GenericProps {
     /** Set panel open or not. */
     isOpen?: boolean;
 
-    /** The function called on open. */
-    openCallback?: Callback;
-
     /** The function called on close. */
-    closeCallback?: Callback;
-
-    /** The function called on open or close. */
-    toggleCallback?(shouldOpen: boolean): void;
```

Comment:
I might have forgotten to keep a `onOpen` callback in my props analysis. The expansion panel contains a trigger (the header) that needs to open it.
In that sens it differs from the dropdown, the dialog and the lightbox that need an external trigger
Maybe we could keep these three callback but named `onClode`, `onOpen` and `onToggleOpen` (for convenience)?

Reply:
That confirms what I thought. I didn't know how else you expected to expand the panel. I'll fix it.